### DISCUSSION TITLE
create-next-app not working behind corporate proxy

### DISCUSSION THREAD

**Discussion by Charlydt** [vercel/next.js#23897] [Help]
I have a corporate proxy and I can't get npx create-next-app to work with any of the "--example example" I get the following error: 

```
"Could not locate an example named "with-typescript". Please check your spelling and try again."
```

Notes:
1. I have set up the following npm configs to work with my proxy
npm config set proxy proxy:port
npm config set https.proxy proxy:port
npm config set https-proxy proxy:port
2. npx create-next-app works fine
3. I have tried different examples and none of them work
4. I have also tried npx create-next-app ts --use-npm --example https://github.com/vercel/next.js/tree/canary/examples/with-typescript
I get following error: Could not locate the repository for "https://github.com/vercel/next.js/tree/canary/examples/with-typescript". Please check that the repository exists and try again.
5. I have also tried yarn and get same errors
6. If I disable the proxy and connect from home works fine

Is there any setting or configuration I am missing to get the examples to work with my proxy? I'm sorry if this is not the right place to report this issue but I'm running out of options

If I disable the proxy, npx --example works just fine

### To Reproduce

1. npx create-next-app --example with-typescript

**Comment by aabuhijleh** [comment]
This [PR](https://github.com/vercel/next.js/pull/34691) attempts to fix this issue.

**Comment by asalamintel** [comment]
I am facing the same issue where npx create-next-app nextjs-blog --use-npm --example "https://github.com/vercel/next-learn-starter/tree/master/learn-starter" returns the following error, for example where it works fine on the home network. 
"Could not locate the repository for "https://github.com/vercel/next-learn-starter/tree/master/learn-starter". Please check that the repository exists and try again."

npm on it's own works just fine.
What does create-next-app use to download the template? Perhaps we can configure that to use proxy?
FWIW, like Charlydt, I have tried adding proxy configuration to .npmrc and passing --proxy all in vain.

Is there any updates on this?

**Reply by johnschult** [reply]
The PR that @aabuhijleh referenced has an out of date branch and isn't a candidate for merging. There will need to be a new PR.
